packages/edit-navigation/src/components/editor/style.scss

```diff
@@ -55,13 +57,13 @@
 
 			// Menu items.
 			// This needs high specificity to override inherited values.
-			&.wp-block-navigation-item.wp-block-navigation-item {
+			&.wp-block-navigation-item.wp-block-navigation-item,
+			&.wp-block-navigation-submenu .wp-block-navigation-submenu {
```

Comment:
This one shouldn't be needed either.

Reply:
Discovered I could remove the whole rule, it didn't seem to do anything as margin is set by a line further up 🤷